apps/blockchain-mailer/tsconfig.json

```diff
@@ -5,8 +5,6 @@
   "compilerOptions": {
     "noImplicitReturns": true,
     "noUnusedLocals": true,
-    "strictNullChecks": true,
-    "strict": true,
```

Comment:
It's fine to remove these now, but ideally we'd add them back in once the project development is stable.

Reply:
In my experience, it's really painful to use TypeScript `strict` mode. I'm not sure it's worth or not, I'll take it a try.